Silicon/Intel/IntelSiliconPkg/Feature/VTd/IntelVTdDmarPei/IntelVTdDmarPei.c

```diff
@@ -514,7 +515,9 @@ InitVTdInfo (
   ASSERT (VTdInfo != NULL);
 
   AcpiDmarTable = GetAcpiDmarTable ();
-  ASSERT (AcpiDmarTable != NULL);
+    ASSERT (AcpiDmarTable != NULL);

```

Comment:
I believe this is missing an "If" statement to check the AcpiDmarTable pointer.

Reply:
Thanks, it's updated. I missed that porting it over from another codebase.